helm/flink-kubernetes-operator/values.yaml

```diff
@@ -154,6 +154,9 @@ defaultConfiguration:
   # If set to false, loads just the overrides as in (2).
   # This option has not effect, if create is equal to false.
   append: true
+  # If set to true, then will support YAML 1.2 syntax through use of `config.yaml` file.
+  # If set to false, will make use of deprecated flink-conf.yaml filen.
+  standardYaml: false
```

Comment:
Would be great if this would be "automatic" based on whether the user specified flink-conf.yaml vs conf.yaml. Do you think that's possible?